types/co-util/tslint.json

```diff
@@ -0,0 +1,10 @@
+{ "extends": "@definitelytyped/dtslint/dt.json",
+    "rules": {
+        "array-type": false,
```

Comment:
It doesn't look like disabling these rules is strictly necessary. Could you instead remove these rules and fix the lint errors in your declaration files? If you have a problem with how to fix a specific lint error, please let me know and I can try to help.